README.md

```diff
@@ -1,140 +1,32 @@
 ecTrans
-*******
+=======
+
+ecTrans is a library for performing efficient and scalable spectral transformations. It is used for transforming fields from a grid point space on the sphere (e.g. latitude-longitude) to a spectral space based on spherical harmonics (for global transformations) or bifourier harmonics (for limited area transformations), which constitutes a direct transform. A corresponding inverse transform can also be performed. A transform consists of a Fourier transform in the longitudinal direction and either a Legendre transform (global) or another Fourier transform (limited area) in the latitudinal direction. ecTrans can also operate on decomposed fields which are partitioned across separate MPI tasks and performs the necessary communication to ensure all data needed for a particular transform are resident on a local task.
```

Comment:
would "distributed" be better than "decomposed"?

Reply:
Yes!